typo.py

```diff
@@ -25,7 +25,7 @@
 def catch_all(path):
     redirect_to = get_best_match(request.url)
     new_url = replace_host(request.url, redirect_to)
-    return redirect(new_url)
+    return redirect(new_url + '?redirected=true')
```

Comment:
You should not pass params as query parms, instead you should .....
```suggestion
    return redirect(new_url + '#redirected=true')
```